src/System.Net.NameResolution/src/System.Net.NameResolution.csproj

```diff
@@ -6,9 +6,6 @@
     <ProjectGuid>{1714448C-211E-48C1-8B7E-4EE667D336A1}</ProjectGuid>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>   
-  <PropertyGroup>
-    <DefineConstants Condition="'$(TargetGroup)' == 'uap'">$(DefineConstants);uap</DefineConstants>
```

Comment:
There is nothing in the code rely on it (no #if uap statement), and there is no reference to UWP.